google/cloud/security/scanner/audit/instance_network_interface_rules_engine.py

```diff
@@ -0,0 +1,251 @@
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Rules engine for NetworkInterface."""
+from collections import namedtuple
+import itertools
+import re
+
+# pylint: enable=line-too-long
+from google.cloud.security.common.util import log_util
+from google.cloud.security.scanner.audit import base_rules_engine as bre
+from google.cloud.security.scanner.audit import errors as audit_errors
+
+LOGGER = log_util.get_logger(__name__)
+
+
+def escape_and_globify(pattern_string):
```

Comment:
Could you use this utility? https://github.com/GoogleCloudPlatform/forseti-security/blob/0adcddda57d79aeaa3866759f35cc0748644d627/google/cloud/security/common/util/regex_util.py#L19